code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -48,7 +48,7 @@
 
 	var/momentum_x = 0		//Realistic physics. No more "Instant stopping while barreling down a hallway at Mach 1".
 	var/momentum_y = 0
-	var/momentum_max = 500
+	var/momentum_max = 250
 	var/momentum_impact_coeff = 0.5	//At this speed you'll start coliding with people resulting in momentum loss and them being knocked back, but no injuries or knockdowns
 	var/momentum_impact_loss = 50
```

Comment:
May want to halve this as well, otherwise you're losing 20% maximum momentum instead of 10%